projectile.el

```diff
@@ -4582,9 +4582,39 @@ The command actually run is returned."
                                                         project-root))))
     (unless (file-directory-p default-directory)
       (mkdir default-directory))
-    (projectile-run-compilation command)
+    (projectile-run-compilation command use-comint-mode)
     command))
 
+(defcustom projectile-configure-use-comint-mode nil
+  "If non-nil, make the output buffer of projectile-configure-project
+interactive."
+  :type 'bool)
```

Comment:
There should be a few other properties here. Refer to existing `defcustoms` for those.